lxdock/cli/main.py

```diff
@@ -74,11 +74,16 @@ def __init__(self, argv=None):
 
         # Creates the 'shell' action.
         self._parsers['shell'] = subparsers.add_parser(
-            'shell', help='Open a shell in a container.',
-            description='Open an interactive shell inside a specific container.')
+            'shell', help='Open a shell or execute a command in a container.',
+            description='Open an interactive shell inside a specific container. If a command is '
+                        'specified, execute the command instead.',
+            usage='lxdock shell [-h] [-u USERNAME] [name] [-c ...]')
         self._parsers['shell'].add_argument('name', nargs='?', help='Container name.')
         self._parsers['shell'].add_argument(
             '-u', '--username', help='Username to login as.')
+        self._parsers['shell'].add_argument(
+            '-c', '--command', nargs=argparse.REMAINDER, dest='commands',
```

Comment:
@hsoft  Yes, you're right. We're using `Host.run(cmd_args)` and `Guest.run(cmd_args)` in the code. I'll change to `cmd_args`. Thank you!